### PR TITLE
knob to enable / disable auto mlag in eos_l3ls_evpn

### LINKED ISSUE #364: To support none-mlag l3leaf nodes under same node-group
**Issue Type**

- Role enhancement

**Is your feature request related to a problem? Please describe.**
Right now the design expects mlag interface with two l3leaf nodes under same node-group.
Hopefully this restriction can be lift to support GW peers, or evpn all-active peers.

**Describe the solution you'd like**
Add a nomlag tag under node-groups
ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/evpn-fabric-l3leaf-yml.j2
---   {%             if ((loop.length == 2) %}
+++ {%             if ((loop.length == 2) and (l3leaf.node_groups[l3leaf_node_group].nomlag is undefined)) %}

### PR DESCRIPTION
**Pull request by carlbuchmann**
## Change Summary

Provide a knob to enable/disable mlag in eos_l3ls_evpn

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Documentation content changes
- [ ] Other (please describe):

## Related Issue(s)

Fixes #364 

## Proposed changes

eos_l3ls_evpn

## How to test

```yaml
l3leaf:
  defaults:
    mlag: false | true
  node_groups:
    <node_group>:
      mlag: false | true

l2leaf:
  defaults:
    mlag: false | true
  node_groups:
    <node_group>:
      mlag: false | true
```

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://www.avd.sh/docs/contributing/) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed ([`pre-commit`](https://www.avd.sh/docs/installation/development/#python-virtual-environment), `make linting` and `make sanity-lint`).
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_l3ls_evpn/README.md [hunk 1]
```diff
@@ -678,6 +682,9 @@ l2leaf:
     # Point-to-Point interface speed - will apply to L2 Leaf and L3 Leaf switches | Optional.
     p2p_link_interface_speed: < interface_speed >
 
+    # Enable / Disable auto MLAG, when two nodes are defined in node group.
```
Comment (titom73) [inline ansible_collections/arista/avd/roles/eos_l3ls_evpn/README.md]: Should not we support this per node_group as well? Would be helpful in a mixed environment or even during design migration with MLAG and EVPN A/A

Comment (titom73) [reply ansible_collections/arista/avd/roles/eos_l3ls_evpn/README.md]: Seems to be supported as per line 75 in `evpn-fabric-l3leaf-yml.j2`

Comment (carlbuchmann) [reply ansible_collections/arista/avd/roles/eos_l3ls_evpn/README.md]: that's correct, maybe the documentation isn't clear, but all variables in `defaults` are also supported per node.
We have the following text on line 549 + 702 of README.md
```yaml
# All variables defined under `defaults` dictionary can be defined under each node group to override it.
```

**Review by titom73 — APPROVED** [review]
LGTM!

**Pull request merged**